build/debian-hyperkube-base/Dockerfile

```diff
@@ -14,14 +14,13 @@
 
 FROM BASEIMAGE
 
-# TODO(#69896): deprecate the shortened aliases in /
-RUN ln -s /hyperkube /apiserver \
+RUN ln -s /hyperkube /kube-apiserver \
  && ln -s /hyperkube /cloud-controller-manager \
- && ln -s /hyperkube /controller-manager \
```

Comment:
did we actually announce these were getting deprecated and removed anywhere? I'd expect at least one release of overlap where both the deprecated and standard names worked